_content/setup-wallets-commandline.md

```diff
@@ -0,0 +1,141 @@
+        "settleThreshold": null,
+        "settleTo": "0"
+    },
+    "rateLimitSettings": {
+        "maxPacketsPerSecond": null
+    },
+    "settlementEngineDetails": null,
+    "customSettings": {
+        "ilpOverHttp.outgoing.url": "https://money.ilpv4.dev/ilp",
+        "ilpOverHttp.incoming.auth_type": "SIMPLE",
+        "ilpOverHttp.incoming.simple.auth_token": "JQBzzLkbslrsT",
+        "ilpOverHttp.outgoing.simple.auth_token": "enc:jks:crypto/crypto.p12:secret0:1:aes_gcm:AAAADDBa7b-nDvY2ydWysQQMoZL7yIOmK-7-3kLiMc9pxhzPw1Ei68OpwcZu6W-j",
+        "ilpOverHttp.outgoing.auth_type": "SIMPLE"
+    },
+    "paymentPointer": "$money.ilpv4.dev/user_qqGV8nij",
+    "parentAccount": false,
+    "childAccount": true,
+    "peerAccount": false,
+    "peerOrParentAccount": false
+}
+```
+
+### 2. Add test XRP to your account
+
+The Testnet has a rainmaker account that you can use to populate your account with test XRP, so you can then use this account to send test XRP to other accounts. To add test XRP to your account, use the following command. Make sure to replace `{your-account-id}` with your own `accountId` created in Step 1.
+
+```
+> curl --location --request POST \
+  'https://hermes-rest.ilpv4.dev/accounts/{your-account-id}/money'
+```
+
+### 3. Check your account balance
+
+To see how much money is in your account, try the following call. Make sure to replace **`{auth_token}`** above with the auth token returned when you generated your account.
+
+```
+> curl --location --request GET 'https://hermes-rest.ilpv4.dev/accounts/{your-account-id}/balance' \
+--header 'Accept: application/json' \
+--header 'Authorization: Bearer {auth_token}' \
+```
+
+This request will return a JSON response similar to the following:
+
+```text
+{
+    "assetCode": "XRP",
+    "assetScale": "6",
+    "accountBalance": {
+        "accountId": "user_qqGV8nij",
+        "netBalance": "0",
+        "clearingBalance": "0",
+        "prepaidAmount": "0"
+    }
+}
+```
+
+### 4. Send test XRP to another payment pointer
+
+You can use your account to send test XRP to a different payment pointer. This example uses a destination payment pointer on [https://rafiki.money](https://rafiki.money). Substitute your `accountId` from Step 1. You can change the destination payment pointer if you prefer.
+
+```
+> curl --location --request POST 'https://hermes-rest.ilpv4.dev/accounts/{your-account-id}/pay' \
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--header 'Authorization: Bearer {auth_token}' \
+--data-raw '{
+  "amount": "10",
+  "destinationPaymentPointer": "$rafiki.money/p/dfuelling"
+}'
+```
+
+This request returns a JSON response similar to the following:
+
+```text
+{
+    "originalAmount": "10",
+    "amountDelivered": "10",
+    "amountSent": "10",
+    "successfulPayment": true
+}
+```
+
+- **`originalAmount`** is the amount that you wanted to send.
+
+- **`amountDelivered`** is the amount your friend actually received.
```

Comment:
Might be worth noting that (I believe) this is denominated in the recipient's asset and units, whereas "amount sent" is in the asset and units of the sender -- these can be different